### DISCUSSION TITLE
Look here for direction on Discussions!

### DISCUSSION THREAD

**Discussion by doyled-it** [mitrefireline/simfire#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to discuss questions and ideas with the community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Discuss feature requests.
  * Don't put a question about the codebase as an issue, start a discussion instead!
 
I'm trying to follow [furo](https://github.com/pradyunsg/furo)'s example on discussions vs. issues. We'll see how it goes!

**Comment by ryanguilherme** [comment]
Hi, does your simulator consider factors like terrain angle, wind speed / direction, oxygen, etc? I'm considering using it on a college work about wildifires, but i need to consider these factors.
